kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -1,57 +1,68 @@
 <template>
-  <div v-show="navShown">
+
+  <div>
     <div
-      class='nav-wrapper'
+      v-show="navShown"
+      class="nav-wrapper"
       :style="wrapperStyle">
-      <div class='header' :style="{ height: headerHeight + 'px', textAlign: !(mobile | tablet) ? 'center' : 'inherit' }">
+      <div class="header"
+           :style="{ height: headerHeight + 'px', textAlign: !(mobile | tablet) ? 'center' : 'inherit' }">
         <button
           v-if="mobile | tablet"
           :aria-label="closeNav"
-          class='close' @click="toggleNav"
+          class="close" @click="toggleNav"
           :style="{ fontSize: headerHeight/2 + 'px' }">
           <ui-icon :icon="mobile ? 'arrow_back' : 'menu'"/>
         </button>
         <img
-          class='logo'
-          v-if="mobile"
+          class="logo"
+          v-else-if="mobile"
```

Comment:
I don't understand this, it's a condition that will never occur, because it will only go to 'else' if neither `mobile` or `tablet` are true.

This is preventing the logo displaying on the mobile side nav.